wiki/Multi/en.md

```diff
@@ -0,0 +1,422 @@
+
+It is entirely possible to start a match with one colour team only, and there is no handicap bonus for the other team with lesser members.
+Placement of the players prior to the match does not matter; all blue members will be grouped to blue team and all red members will be grouped to red team.
+There is no role (like "Leader" or "Captain") distinction in the team; compete for both the team top spot and beating the other team score in total as a team.
+
+The surviving highest scoring member of either team will be shown as 1st at their respective team.
+Members in failed state will be located at the bottom of the team leaderboard and will not contribute to the team total as long as the members are in failed state.
+
+![Team Vs Grade screen][Team Vs Grade image]
+
+_Team Vs grade screen_
+
+<!-- Result -->
+In the results screen, declaration of match winner will be based on team total and announced at the center in winner's colour, with blue team total at the left and red team total at the right.
+Only surviving members of the team's score will be counted; it is possible for red team to be declared winner of the match when blue team has the superiority just because some blue team members providing the superiority were in failed state.
+
+The Team tab will show the total amount of hit score (300, 100, etc) done by the whole team and the average of the accuracy of the team.
+Combo not counted.
+Grade will not be shown on "Team" tab but will be shown at player's tab as watermark (if passed).
+
+### Tag Coop / Tag Team Vs
+
+_Only available in osu! mode. **UNRANKED**_
+
+![Tag Coop Interface][Tag Coop Interface image]
+
+_Tag Coop interface_
+
+![Tag Team Vs Interface][Tag Team Vs Interface image]
+
+_Tag Team Vs interface_
+
+<!-- Interface -->
+Tag Coop let everyone in the Match Setup work together to pass the beatmap, one combo at a time.
+Tag Team Vs is just Tag Coop with red/blue team competition.
+
+Do note that this team mode is only available in **_osu!_ only**, and it is an **UNRANKED** play (meaning, it will not be counted to ranked score).
+
+In this mode, player placement is important as it will determine when it is the player's turn.
+All players shares a lifebar / Each team has its own respective lifebar that are visible to teammates only.
+Unlike other team modes, failing the beatmap means returning to the Match Setup immediately without results / giving victory to the other team.
+
+An arrow will show the player who is currently playing the beatmap's combo.
+The name will be shown at the lower right of the screen.
+There will be warning arrows when it is the player's turn and the combo will be in colour until the combo ends.
+The player can adjust for personal combo colour at Match Setup.
+
+**All players must do the spinner**.
+Refusal or unable to complete the spinner will break the current combo count.
+
+If a player quits, [Auto][Auto wikilink] will take over the "Quit" player.
```

Comment:
rephrase to: `If a player quits during the match, [Auto][Auto wikilink] will take over for the player that left.`